static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.tsx

```diff
@@ -111,19 +110,31 @@ export function TimeSeriesWidgetVisualization(props: TimeSeriesWidgetVisualizati
   const organization = useOrganization();
   const navigate = useNavigate();
 
-  let releaseSeries: Series | undefined = undefined;
-  if (props.releases) {
-    const onClick = (release: Release) => {
-      navigate(
-        makeReleasesPathname({
-          organization,
-          path: `/${encodeURIComponent(release.version)}/`,
-        })
-      );
-    };
-
-    releaseSeries = ReleaseSeries(theme, props.releases, onClick, utc ?? false);
-  }
+  const releaseDataSeries =
+    props.releases &&
+    ReleaseSeries(
+      theme,
+      props.releases,
+      function onReleaseClick(release: Release) {
+        navigate(
+          makeReleasesPathname({
+            organization,
+            path: `/${encodeURIComponent(release.version)}/`,
+          })
+        );
+      },
+      utc ?? false
+    );
+
+  const releaseSeries = releaseDataSeries
+    ? [
+        LineSeries({
+          ...releaseDataSeries,
+          name: releaseDataSeries.seriesName,
+          data: [],
+        }),
```

Comment:
I feel like `ReleaseSeries` should handle this, i'm actually not even sure if this is needed since it's not a line series at all. The release lines are   mark lines

Reply:
That tracks